tilavarauspalvelu/api/graphql/types/reservation/serializers/create_serializer.py

```diff
@@ -86,6 +90,7 @@ def validate(self, data: ReservationCreateData) -> ReservationCreateData:
         data["unit_price"] = pricing.highest_price
         data["tax_percentage_value"] = pricing.tax_percentage.value
         data["non_subsidised_price"] = data["price"]
+        data["access_type"] = AccessType(reservation_unit.access_type)
```

Comment:
get_access_type_at(reservation.begins)

Reply:
Right, forgot about this, will add with tests

Reply:
Fixed.